lib/caesar.rb

```diff
@@ -1,6 +1,24 @@
+require "pry"
 class Caesar
-  attr_reader :key
+
 
   def initialize
+
+  end
+
+
+  def caesar_cipher(string, shift)
+    array = Array.new
+    array << string.chars
```

Comment:
string.chars creates an array. No need to shovel this into a new array. In fact, this will result in a nested array i.e.
```
string = "hello"
array = Array.new
array << string.chars
=> [["h", "e", "l", "l", "o"]]
```